helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/constraints/SoftConstraint.java

```diff
@@ -25,46 +25,29 @@
 
 /**
  * The "soft" constraint evaluates the optimality of an assignment by giving it a score of a scale
- * of [minScore, maxScore]
+ * of (0, 1]
  * The higher the score, the better the assignment; Intuitively, the assignment is encouraged.
  * The lower score the score, the worse the assignment; Intuitively, the assignment is penalized.
  */
 abstract class SoftConstraint {
-  private float _maxScore = 1000f;
-  private float _minScore = -1000f;
+  // The hyper parameter in the normalize function; Child class can override and customize the value
+  private float alpha = 1.0f;
```

Comment:
As we discussed, the alpha might be determined by the cluster statue, right? For example, depends on the estimated partition count. Or you suggest we don't estimate but always use the fixed alpha?
Please comment on your plan.

Reply:
We can use fixed alpha =  1.0f at beginning, making it a private field gives us the ability to set the value from constructor if in the future we have the idea of how to tune it.